frontend/src/components/Loader/index.tsx

```diff
@@ -1,5 +1,30 @@
-const Loader = () => {
-    return <div>Loader</div>;
+import React from "react";
+import { Box, CircularProgress, Typography } from "@mui/material";
+
+type LoadingProps = {
+  size?: number;
+  message?: string;
+};
+
+const Loading: React.FC<LoadingProps> = ({ size, message }) => {
+  return (
+    <Box
+      sx={{
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+        flexDirection: "column",
+        height: "100vh",
+      }}
```

Comment:
avoid inline styles, rather use styled components of MUI.

Reply:
Provide the example